docs/release-notes/2435-fix-resource-pool-not-saved.txt

```diff
@@ -0,0 +1,5 @@
+:orphan:
```

Comment:
I don't know if the release note is necessary; it seems to me like the bug was introduced by expconf, since the old code path updated `expModel.Config.Resources` directly.

Reply:
I think this is necessary since it's a bug introduced before the last release. The WebUI doesn't show the resource pool if it is filled automatically.

Reply:
I just checked.  This is not a bug on 0.15.5.

See https://github.com/determined-ai/determined/commit/afdb74955b5ec498095767b1d81d81756b6da510#diff-d92b0050d5549a1efe86c524037f8a2709d6ebf8ce2c63a777f63744bc8a0b00R117